FreeRTOS/iot-reference-arm-corstone3xx
# FRI Release `v202403.00`

FRI Release `v202403.00`

Description
-----------


* Added ML reference applications
    * Keyword Detection
    * Speech Recognition
    * Object Detection
* Build and debug reference applications using VS Code on:
    * Linux
        * Ubuntu 20.04
    * Windows 11
    * Mac OS 14.x
* Device provisioning support
  * Create device credentials (certificate and private key) using python script
    and provision them into the device.
* Validated keyword-detection application using AWS Device Advisor tests
* Removed aws-iot-example reference application
* Update to ml-eval-kit version 23.11
* Updated to Mbed TLS v3.5.2
* Updated to TF-M v2.0.0
* Re-structure the project to ease maintenance

Test Steps
-----------
Validated in the CI and no regression observed.

Checklist:
----------



Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
